src/web/layouts/StandardLayout.tsx

```diff
@@ -129,6 +129,12 @@ export const StandardLayout = ({ CAPI, config, NAV }: Props) => {
                     </ArticleContainer>
                     <ArticleRight>
                         <StickyAd config={config} />
+                        <Section
```

Comment:
Do we need a `Section` here? I suspect you can just render the component directly

Reply:
In other places we just use a plain div as the root. E.g.:

```
<div data-island="most-viewed-list" />
```

Reply:
Done